sdk/nodejs/invoke.ts

```diff
@@ -34,10 +34,4 @@ export interface InvokeOptions {
      * invoke.
      */
     version?: string;
-
-    /**
-     * Invoke this function asynchronously.  If 'true' is passed in here, then an invoked function
-     * will only supply the `Promise<>` side of its result.
-     */
-    async?: boolean;
```

Comment:
note: we could leave this in to ensure that TS doesn't err.

Reply:
I think we need to keep this - we are about to push all code to require passing this - we don't want that to be a hard break - we just want to change the default value of this.

Reply:
WFM.

Reply:
Have kept this in.  #resolved